Tests/Shared.Specs/Specialized/TaskCompletionSourceAssertionSpecs.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions.Extensions;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs
+{
+    public class TaskCompletionSourceAssertionSpecs
```

Comment:
Could you add a test for the `subject is object` check?

Reply:
Hope, the new test is what you mean.

Reply:
It was 👍